<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="aspectBucketingCbx">
     <property name="text">
      <string>Aspect Ratio Bucketing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="latentCachingCbx">
     <property name="text">
      <string>Latent Caching</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clearCacheCbx">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Clear Cache Before Training</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>latentCachingCbx</sender>
   <signal>toggled(bool)</signal>
   <receiver>clearCacheCbx</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
